fdbserver/include/fdbserver/DataDistributionTeam.h

```diff
@@ -69,9 +69,41 @@ FDB_DECLARE_BOOLEAN_PARAM(ForReadBalance);
 FDB_DECLARE_BOOLEAN_PARAM(PreferLowerReadUtil);
 FDB_DECLARE_BOOLEAN_PARAM(FindTeamByServers);
 
+class TeamSelect {
+public:
+	enum Value : int8_t {
+		INVALID = 0,
+		AVOIDMOVEMENT, // Avoid data movement by selecting srcTeam if it's not busy
+		WANTSRCSERVERS, // Ask for srcTeam
+		WANTTRUEBEST, // Ask for the most or least utilized team
+		OTHER
+	};
+	TeamSelect() : value(INVALID) {}
```

Comment:
Can we get away without having the default `INVALID` value (or the default `TeamSelect()` constructor)? That'll be cleaner and less error-prone since `INVALID` is a non-actionable value.

Reply:
If we need a default, then would it make sense to have `OTHER` (or `ANY`) be the default?

Reply:
Yes, `INVALID` does seem annoyed. I've moved out `INVALID` and make `ANY` for default constructor.